src/commons/src/utils.rs

```diff
@@ -3,6 +3,12 @@ use candid::{decode_one, encode_args, Nat, Principal};
 use pocket_ic::PocketIc;
 use pool_canister::{Account, TransferArg};
 
+// Constants
+lazy_static::lazy_static! {
+    pub static ref ALLOWED_ERROR: Nat = Nat::from(10u64);
+    pub static ref TRANSFER_FEE: Nat = Nat::from(10u64);
```

Comment:
Looks like you can import TRANSFER_FEE from pool canister itself, use it instead of creating another constant with the same name.